### PR TITLE
configure Travis CI

### PR DESCRIPTION
**Pull request by lexidor**

### CODE REVIEW

**Comment by lexidor** [comment]
image
I can't turn on travis for the quizlet one. According to @azjezz that requires someone with write access. The files I can supply, however.

**Comment by lexidor** [comment]
Fixes #9

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+sudo: required
+language: generic
```
Comment (azjezz) [inline .travis.yml]:
```suggestion
language: shell
```

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+sudo: required
```
Comment (azjezz) [inline .travis.yml]: sudo is deprecated and has no effects

**Review by azjezz — CHANGES_REQUESTED** [review]

**Comment by tyronjung-quizlet** [comment]
Thanks for your help @azjezz!

**Pull request merged**